Cargo.toml

```
@@ -58,8 +65,24 @@ tokio-stream = { version = "0.1", features = ["sync"] }
 
 graphql-ws-client.path = "."
 
-graphql-ws-client.features = ["client-cynic", "client-graphql-client", "tungstenite"]
+graphql-ws-client.features = [
+    "client-cynic",
+    "client-graphql-client",
+    "tungstenite",
+]
 
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[workspace.lints.rust]
+unsafe_code = "forbid"
+
+[workspace.lints.clippy]
+pedantic = { level = "warn", priority = -1 }
+cargo = { level = "warn", priority = -1 }
+module_name_repetitions = "allow"
+multiple_crate_versions = "allow"
```

I am usually happy with clippys defaults.  I _definitely_ don't want `pedantic` warnings.

The cargo ones I'd be willing to consider, although `multiple_crate_versions` looks like one of the more useful ones.  What error is it giving if you don't have that allowed?

```suggestion
[workspace.lints.clippy]
cargo = { level = "warn", priority = -1 }
```

---

Fixed